src/utils.test.ts

```diff
@@ -7,10 +7,14 @@ import {
   uriHandshakePrefix,
   uriHandshakeResolverPrefix,
   getFileMimeType,
+  convertSkylinkToBase32,
+  addSubdomain,
 } from "./utils";
 
 const portalUrl = defaultSkynetPortalUrl;
 const skylink = "XABvi7JtJbQSMAcDwnUnmp2FKDPjg8_tTTFP4BwMSxVdEg";
+const skylinkBase32 = "bg06v2tidkir84hg0s1s4t97jaeoaa1jse1svrad657u070c9calq4g";
+const portalUrlSubdomain = addSubdomain(portalUrl, skylinkBase32);
```

Comment:
this should be hardcoded string, in case `addSubdomain` malfunctions, we will not be able to pinpoint the issue here

```suggestion
const portalUrlSubdomain = `https://${skylinkBase32}.siasky.net`;
```